test/config-luatest/rpc_test.lua

```diff
@@ -15,6 +16,24 @@ local function skip_if_no_vshard()
     t.skip_if(not has_vshard, 'Module "vshard-ee/vshard" is not available')
 end
 
+local function start_stub_servers(g, dir, instances)
+    g.stub_servers = g.stub_servers or {}
+    for _, instance in ipairs(instances) do
+        local uri = ('%s/%s.iproto'):format(dir, instance)
+        local s = socket.tcp_server('unix/', uri, function(s)
+            s:detach()
+        end)
+        t.assert(s)
+        g.stub_servers[instance] = s
+    end
+end
+
+local function stop_stub_servers(g)
+    for _, server in pairs(g.stub_servers) do
+        server:close()
```

Comment:
Found out this may error if the socket is already closed. I should check if the socket is open before closing it.

Reply:
Fixed by removing the socket "detach".